students/guillaume/session03/list_lab.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
you are getting quite ahead of yourself (or the rest of us, anyway) nothing wrong with that, but as this is a session 3 exercise -- we will review it then. (any maybe put it in a "session 3" dir to keep things straight.

I do apologize -- the first few sessions are going to feel slow for folks that have some Python experience. But not everyone is the class does have much experience with python, so we spend the first few classes on the basics to get everyone on the same page.